### PR TITLE
feat(networking): create vpc and public subnets

### PR DESCRIPTION
**Pull request by benhur-sre**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Ignore Terraform state files
```
Comment (carlosgit2016) [inline .gitignore:1]: You could use https://github.com/github/gitignore/blob/main/Terraform.gitignore

File: variables-locals.tf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+locals {
```
Comment (carlosgit2016) [inline variables-locals.tf:1]: Why the `variables-locals.tf` ? 

This deviate from terraform standard structure https://developer.hashicorp.com/terraform/language/modules/develop/structure

File: variables-locals.tf [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  availability_zones = {
+    az1 = "eu-north-1a"
```
Comment (carlosgit2016) [inline variables-locals.tf:3]: is Europe cheaper ?

File: variables-locals.tf (hunk 2 above)
Comment (carlosgit2016) [inline variables-locals.tf:1]:
```suggestion
locals {
  project = "platform-master"
```

File: versions.tf [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  }
+}
+
+provider "aws" {
+  region = "eu-north-1"
+  default_tags {
+    tags = {
+      project = var.project
+      onwer= "Benhur A. Silva"
+      owner-linkedin = "linkedin.com/in/benhuraraujo/"
```
Comment (carlosgit2016) [inline versions.tf:23]: :+1:

File: variables-locals.tf (hunk 3 above)
Comment (carlosgit2016) [reply variables-locals.tf:3]: I saw the comment in Discord, nvm then

File: .gitignore (hunk 1 above)
Comment (benhur-sre) [reply .gitignore:1]: nice repo

File: variables-locals.tf (hunk 2 above)
Comment (benhur-sre) [reply variables-locals.tf:1]: I liked. But I like to have locals separated. I'll create a specific file for them

**Review by carlosgit2016 — APPROVED** [review]

**Pull request merged**